common/src/main/java/io/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -137,7 +137,8 @@
      * @return the s
      * @throws EnhancedServiceNotFoundException the enhanced service not found exception
      */
-    public static <S> S load(Class<S> service, String activateName, Class[] argsType, Object[] args) throws EnhancedServiceNotFoundException {
+    public static <S> S load(Class<S> service, String activateName, Class[] argsType, Object[] args)
```

Comment:
Why change the line here ？
I think we should have more characters on each line to avoid more line breaks.

Reply:
hard wrap at 120 cols
![image](https://user-images.githubusercontent.com/8758457/58403438-448bb600-8095-11e9-98b1-24d6d217ab76.png)